packages/client/hmi-client/src/assets/css/theme/_fonts.scss

```
@@ -21,4 +21,48 @@ h5 {
 
 h6 {
     font-size: $fontBodySmall;
-} 
+}
+
+
+/*
+The Latin Modern fonts are released under the GUST font license, which is
+legally equivalent to the LaTeX Project Public License. See the files in this
+directory for details. The fonts can be obtained from several TeX distributions
+or package managers.
+
+The WOFF fonts have been obtained from
+http://www.gust.org.pl/projects/e-foundry/
+*/
+
+@font-face {
+    font-family: 'latin-modern-math';
```

Could we rename it `latex` because that's why we are using it, makes it simpler to keep track when looking at source-code.
```suggestion
    font-family: 'latex';
```

---

done